javascript/javascript1/week3/index.js

```diff
@@ -0,0 +1,244 @@
+//question 1:Item array removal------
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+];
+const nameToRemove = "Ahmad";
+for (let i = 0; i < names.length; i++) {
+    if (names[i] === nameToRemove) {
+        names.splice(i, 1)
+    }
+}
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+//question 2: 8 hours and 38 minutes
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+const travelTime = getTravelTime(travelInformation);
+function getTravelTime() {
+    let result = travelInformation.destinationDistance / travelInformation.speed; //8.64
+    return Math.floor(result) + " hours and " + Math.floor((result - Math.floor(result)) * 60) + " minutes.";
+}
+console.log(travelTime);
+
+//question 3:Series duration of my life---------
+const seriesDurations = [
+    {
+        title: "Run, brothers",
+        days: 27,
+        hours: 12,
+        minutes: 24,
+    },
+    {
+        title: "Movie&TV Awards",
+        days: 43,
+        hours: 10,
+        minutes: 24,
+    },
+    {
+        title: "MTV Videos Music Awards",
+        days: 46,
+        hours: 14,
+        minutes: 24,
+    },
+    {
+        title: "Game of thrones",
+        days: 3,
+        hours: 1,
+        minutes: 0,
+    }
+];
+
+function logOutSeriesText(seriesDurations) {
+    // write code here
+    let string = "";
```

Comment:
string is not a good name for the variable as it's a keyword, maybe result or outputText

Reply:
oh, yes you are right.

Reply:
Hi aina, I have changed all of the issues. pls, check for me again.